### ISSUE TITLE
YSmenu Theme

### ISSUE THREAD

**Issue by DraconicNEO** [DS-Homebrew/TWiLightMenu#1605]
**Is your feature request related to a problem? Please describe.**
My idea is to have an option for using a YSmenu theme in Twilight menu, similar to how we have different Menu themes for DSi, R4, Sega CD, etc. 

**Describe the solution you'd like**
A new option for a YSmenu theme in the settings. Ideally this new YSmenu theme should be compatible with Actual YSmenu themes, although unlike YSmenu it whould have the individual YSmenu themes contained in separate nameable folders so the user can select between them easier (The original YSmenu doesn't have theme configuration options, files had to be moved around to select themes). 

**Describe alternatives you've considered**
N/A

**Additional context**

**Comment by NightScript370** [comment]
Isn't this basically the R4 theme?

**Comment by lifehackerhansol** [comment]
Not quite. The intent of this request is to allow the regular YSMenu configurations for theming be supported, which is _completely_ incompatible with the R4 menu at the moment. It would either have to be a new theme or a rewrite of the existing one.
